### PR TITLE
UT with one string and one point

### PR DESCRIPTION
**Pull request by LeeGregory12d**
An IFC 4x3 file with data in MGA 56 coordinates and AHD.
The data is one string names and one point.

**ATTENTION:** This template provides structure for 
- a general pull request and 
- a specific pull request in scope of IFC4.3 validation work within IFC Infrastructure Extension Deployment project.
Please fill in the corresponding section(s) and delete the rest.

# General pull request

Fixes # . *reference the issue fixed through this pull request*

## Changes and improvements proposed by this pull request
- *list all functions / files / changes you have made*
- 

*Use `@[username]` to gain attention by potential reviewers.*

# IFC4.3 Unit test

## Unit test
*provide a short description*

## Review requested
- *List all functions/concepts/entities you have added/changed/improved*
- 

@pjanck

## Link to external documentation
*optional*

## Instructions
- provide a descriptive title
> Added new UT <UnitTestName> with supporting documentation
> Expanded UT <UnitTestName> with ...
> Corrected / Amended / Updated ...
- prepend the title with `[UT] <title>` to enable easier management
- mark as WIP if work in progress
> WIP: [UT] <title>
- label the pull request with `infra depl`
- please set at least @pjanck as a reviewer and add others as needed
- delete this section

### CODE REVIEW

File: IFC 4.3/Georeferencing_Tiny_Tin_1/UT_tiny_tin_in_MGA_56.ifc [hunk 1]
```diff
@@ -29,7 +29,7 @@ DATA;
 #22 = IFCMAPCONVERSION(#14, #21, 0., 0., 0., 1., 0., 1.);
 #23 = IFCSITE('2_$LqFwhrElv$jrhYYj9A2', #2, 'Default Site', 'Description of Default Site', $, #15, $, $, .ELEMENT., $, $, $, $, $);
 #24 = IFCRELAGGREGATES('3iLDj9mYb82A6se0A0HaK1', #2, 'ProjectContainer', 'ProjectContainer for Sites', #1, (#23));
-#25 = IFCBUILDING('17MWGb$P5Dn9Ujq7lL$KyC', #2, 'tin tiny', 'Description of tin tiny', $, #15, $, $, .ELEMENT., $, $, $);
+#25 = IFCFACILITY('17MWGb$P5Dn9Ujq7lL$KyC', #2, 'tin tiny', 'Description of tin tiny', $, #15, $, $, .ELEMENT., $, $, $);
```
Comment (pjanck) [inline IFC 4.3/Georeferencing_Tiny_Tin_1/UT_tiny_tin_in_MGA_56.ifc:32]: `IfcFacility` [documentattion](https://standards.buildingsmart.org/IFC/DEV/IFC4_3/RC1/HTML/schema/ifcproductextension/lexical/ifcfacility.htm) has only 9 attributes, and not 12 as the original `IfcBuilding`.

**Review by pjanck — CHANGES_REQUESTED** [review]
Please correct the number of attributes in `IfcFacility`.

File: IFC 4.3/Georeferencing_Tiny_Tin_1/UT_tiny_tin_in_MGA_56.ifc (hunk 1 above)
Comment (pjanck) [reply IFC 4.3/Georeferencing_Tiny_Tin_1/UT_tiny_tin_in_MGA_56.ifc]: Resolved in f8cf19d

**Review by pjanck — APPROVED** [review]

File: IFC 4.3/Georeferencing_Tiny_Survey_1/UT_one_string_one_pont.ifc [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+#44 = IFCCARTESIANPOINT((256395.03, 7011722.26, 45.83));
+#45 = IFCCARTESIANPOINT((256381.65, 7011725.57, 45.91));
+#46 = IFCCARTESIANPOINT((256372.3, 7011727.61, 45.91));
+#47 = IFCCARTESIANPOINT((256368.305104578, 7011728.47766959, 45.91));
+#48 = IFCSURFACESTYLE($, .POSITIVE., (#49));
+#49 = IFCSURFACESTYLERENDERING(#50, $, $, $, $, $, IFCNORMALISEDRATIOMEASURE(3.90625E-3), IFCSPECULAREXPONENT(10.), .NOTDEFINED.);
+#50 = IFCCOLOURRGB('grey', 9.01960784313726E-1, 9.01960784313726E-1, 9.01960784313726E-1);
+#51 = IFCPRESENTATIONSTYLEASSIGNMENT((#48));
+#52 = IFCSTYLEDITEM(#31, (#51), $);
+#53 = IFCBUILDINGELEMENT('18x1jaUq54pBDn0OQObE1f', #2, 'SF LE', 'test', 'Super', #15, #54, $, $);
```
Comment (pjanck) [inline IFC 4.3/Georeferencing_Tiny_Survey_1/UT_one_string_one_pont.ifc:60]: It's `IfcBuiltElement` [documentation](https://standards.buildingsmart.org/IFC/DEV/IFC4_3/RC1/HTML/schema/ifcproductextension/lexical/ifcbuiltelement.htm)

**Review by pjanck — CHANGES_REQUESTED** [review]

File: IFC 4.3/Georeferencing_Tiny_Survey_1/UT_one_string_one_pont.ifc (hunk 2 above)
Comment (LeeGregory12d) [reply IFC 4.3/Georeferencing_Tiny_Survey_1/UT_one_string_one_pont.ifc]: resolved in b97e6fa

**Pull request merged**